### ISSUE TITLE
Integrate Katie's new graphics

### ISSUE THREAD

**Issue by katiesteckles** [MathsJam/mathsjam-site#10]
When they're done

**Comment by christianp** [comment]
The graphics will be inspired by [the colour plates in the back of Georges Papy's "Groups"](https://archive.org/stream/Groups_492/Papy-Groups#page/n135/mode/2up).

**Comment by katiesteckles** [comment]
First go:
https://www.dropbox.com/s/jhj5wgnqotfxi5x/papy_1.png?dl=0

**Comment by christianp** [comment]
Can you put SVG versions somewhere?

**Comment by katiesteckles** [comment]
In this folder!
https://www.dropbox.com/sh/tlm1zwo6h2p8m48/AABVonlHV93mf5a9JkTorVPPa?dl=0

**Comment by christianp** [comment]
OK, I've put that in. I swapped the colours for their negatives so that the screen effect of the main page colour works. I also rotated it and changed the scale, so that the image is repeated 2.5 times across the header, no matter the size.
Here are a few screens:
![image](https://cloud.githubusercontent.com/assets/19513/23012688/4ab091d6-f41f-11e6-9a36-11dfdac9647d.png)
![image](https://cloud.githubusercontent.com/assets/19513/23012731/7dc03dba-f41f-11e6-95cb-9afee7991cda.png)
![image](https://cloud.githubusercontent.com/assets/19513/23012747/931e382e-f41f-11e6-8e10-9c67b1e73e05.png)

It might look better if we combine two of Papy's plates into each header.
